### PR TITLE
HLM | 727 Integrate with React router

### PR DESCRIPTION
**Pull request by vishwajeet-eGov**

### CODE REVIEW

**Comment by vishwajeet-eGov** [comment]
@srprasanna @saiprakash-v Please review.

File: tenant-portal/src/index.js [hunk 1]
```diff
@@ -9,15 +9,15 @@ import keycloak from "./utils/keycloak";
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+ // <React.StrictMode>
```
Comment (saiprakash-v) [inline tenant-portal/src/index.js:12]: @vishwajeet-eGov you forgot uncomment it?

File: tenant-portal/src/utils/keycloak.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import Keycloak from 'keycloak-js';
 
-const keycloak = new Keycloak('/tenant-portal/keycloak.json');
+const keycloak = new Keycloak('/keycloak.json');
```
Comment (saiprakash-v) [inline tenant-portal/src/utils/keycloak.js:3]: When I was testing it after building an image, it wasn't picking this file with the path you provided for keycloak.json. Please check once and confirm if it is working

File: tenant-portal/src/index.js (hunk 1 above)
Comment (vishwajeet-eGov) [reply tenant-portal/src/index.js]: Uncommented

**Review by saiprakash-v — APPROVED** [review]

**Pull request merged**